airflow/ui/src/layouts/Nav/Nav.tsx

```diff
@@ -78,29 +83,33 @@ export const Nav = () => {
           icon={<FiDatabase size="1.75rem" />}
           isDisabled
           title="Assets"
+          to="assets"
         />
         <NavButton
           icon={<FiBarChart2 size="1.75rem" />}
           isDisabled
           title="DAG Runs"
+          to="dag_runs"
         />
         <NavButton
           icon={<FiGlobe size="1.75rem" />}
           isDisabled
           title="Browse"
+          to="browse"
         />
         <NavButton
           icon={<FiSettings size="1.75rem" />}
           isDisabled
           title="Admin"
+          to="admin"
         />
       </Flex>
       <Flex flexDir="column">
         <NavButton
           as={Link}
-          href={import.meta.env.VITE_LEGACY_API_URL}
           icon={<FiCornerUpLeft size="1.5rem" />}
           title="Return to legacy UI"
+          to={import.meta.env.VITE_LEGACY_API_URL}
```

Comment:
Does this still work now? It looks like the `NavButton` was intended to handle this internally but did not include the actual logic.

Reply:
Yes, I made sure to manually test that